src/registry.rs

```diff
@@ -793,6 +900,26 @@ mod test {
     );
   }
 
+  #[test]
+  fn deserializes_pkg_info_with_deprecated_as_array() {
+    let text = r#"{ "version": "1.0.0", "dist": { "tarball": "value", "shasum": "test" }, "dependencies": ["key","value"], "deprecated": ["aa"] }"#;
+    let info: NpmPackageVersionInfo = serde_json::from_str(text).unwrap();
+    assert_eq!(
+      info,
+      NpmPackageVersionInfo {
+        version: Version::parse_from_npm("1.0.0").unwrap(),
+        dist: NpmPackageVersionDistInfo {
+          tarball: "value".to_string(),
+          shasum: "test".to_string(),
+          integrity: None,
+        },
+        dependencies: HashMap::new(),
+        deprecated: Some("aa".to_string()),
```

Comment:
This should return `None`. Can you add another test where `deprecated` field is actually a string?